### DISCUSSION TITLE
Is it possible to have more than 10,000 users in a project?

### DISCUSSION THREAD

**Discussion by Edijae** [appwrite/appwrite#5113] [Q&A] [answered]
### 🔖 Feature description

I'm yet to understand why project users limit is 10,000. what will happen when your app users exceed 10,000 or you are expecting your app users to be in millions? if users limit cannot be increased, is there an alternative apart from creating another Users collection?

### 🎤 Pitch

In my case, once i release the app, i'm expecting users to be more than 10,000

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment, marked as answer]
@Edijae, thanks for creating this! 🙏🏼 

The maximum limit you can set is 10,000, but you can have unlimited users if you change the limit to "Unlimited." This 10,000 maximum limit is in place to prevent trying to count the users in the database when there are hundreds of thousands of records since counting can be an expensive operation as the number of records grows.

**Reply by Edijae** [reply]
@stnguyen90 ooh ok. so then then "The maximum limit is 10,000 users." text label is misleading. it should probably be placed below the "limited" option textbox.  

<img width="1119" alt="Screenshot 2023-02-21 at 12 50 48" src="https://user-images.githubusercontent.com/14346659/220310634-ec8910a8-3b9a-4961-95a6-6e4f36ec5315.png">

**Reply by ItzNotABug** [reply]
I'd really like to see this as I was initially having issues understanding as why is it Unlimited but with a max of 10K users.
The maximum limit can either be shown below `Limited` or when `Limited` is selected.

**Comment by gepd** [comment]
wouldn't be a better option to limit the count? I mean after 10.000 users stop to count users?

**Reply by 3IMAD69** [reply]
> wouldn't be a better option to limit the count? I mean after 10.000 users stop to count users?

HARVARD IS CALLING  🔥🔥
